CHANGE_LOG

```diff
@@ -1,7 +1,34 @@
-v0.32.0dev
+v0.32.0rc1
```

Comment:
Do we normally at the data at RC phase? If we don't add the date now will we remember to do it later?